src/main/flight/pid.c

```diff
@@ -128,7 +129,7 @@ static FAST_RAM_ZERO_INIT float airmodeThrottleOffsetLimit;
 
 #define LAUNCH_CONTROL_YAW_ITERM_LIMIT 50 // yaw iterm windup limit when launch mode is "FULL" (all axes)
 
-PG_REGISTER_ARRAY_WITH_RESET_FN(pidProfile_t, PID_PROFILE_COUNT, pidProfiles, PG_PID_PROFILE, 11);
+PG_REGISTER_ARRAY_WITH_RESET_FN(pidProfile_t, PID_PROFILE_COUNT, pidProfiles, PG_PID_PROFILE, 12);
```

Comment:
I'm not sure if it needs to incremente the version. If you add elements at the end I think is not needed, but better wait for others to confirm that.

Reply:
https://github.com/betaflight/betaflight/commit/f5bc673ada2c82d5749327013ed3b6999104cb8b is the change from 10 to 11. You need to increment the Parameter Group number, so the flight controller resets configuration. 

Otherwise you end up with settings in the wrong spot. As an example when I added an OSD element to the thing I'm working on, I did not increment the parameter group, and so osd_units got corrupted and was neither METRIC nor IMPERIAL for awhile.

Reply:
@McGiverGim : This is true for normal PG's, ARRAY element extension is not handled correctly (additional metadata would be needed and nobody implemented it)